Detections/AzureActivity/AzDiagSettingsDeleted.yaml

```diff
@@ -39,7 +42,7 @@ entityMappings:
     fieldMappings:
```

Comment:
And also update entity mapping to:
entityMappings:
  - entityType: Account
    fieldMappings:
      - identifier: Name
        columnName: Name
      - identifier: UPNSuffix
        columnName: UPNSuffix
      - identifier: AadUserId
        columnName: AadUserId
  - entityType: IP
    fieldMappings:
      - identifier: Address
        columnName: CallerIpAddress

Reply:
Done